cla_public/apps/base/healthchecks.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+
+import os
+import requests
+from flask import current_app
+
+
+def check_disk():
+    stat = os.statvfs(os.getcwd())
+    available_mb = (stat.f_bavail * stat.f_frsize) / (1024.0 ** 2)
+    total_mb = (stat.f_blocks * stat.f_frsize) / (1024.0 ** 2)
+
+    available_percent = available_mb / total_mb * 100
+    status = True if available_percent > 2.0 else False
+
+    return {
+        'status': status,
```

Comment:
I've run this on my local machine and these are the results:

```
{
  "Backend API test": {
    "response": "ConnectionError", 
    "status": false, 
    "url": "http://127.0.0.1:8000/status/healthcheck.json"
  }, 
  "disk": {
    "available_mb": 325536.171875, 
    "available_percent": 68.26059495331863, 
    "status": true, 
    "total_mb": 476902.04296875
  }
}
```

The `status: true` doesn't read well to me. I think something like: 

- `healthy: true` or `healthy: false`
- `status: healthy` or `status: unhealthy`

Reply:
Yeah, it's horrible. I followed the existing schema at the time.
Now that we know that it's only the HTTP response code that matters, we can change it.

Reply:
Is it worth `2.0` be a constant to make it more obvious that it's a threshold?

Reply:
♻️

Reply:
> Is it worth `2.0` be a constant to make it more obvious that it's a threshold?

I feel in this current state it's readable enough to keep it as-is:

```
    status = HEALTHY if available_percent > 2.0 else UNHEALTHY
```

What do you think?